test/testClient.js

```diff
@@ -32,11 +35,11 @@ describe('mercurius clients support', function() {
     request(mercurius.app)
     .get('/devices/' + token)
     .send()
+    .expect(200)
     .expect(function(res) {
-      assert.equal(res.status, 200);
-      assert.equal(res.body.clients.length, 1);
-      assert.equal(res.body.clients.indexOf('clientXZ'), 0);
-      assert.equal(res.body.machines.machineXZ.clients.clientXZ, '1');
+      res.body.clients.length.should.equal(1);
```

Comment:
IIRC there's some way to directly check the length (something like array.should.have.length).